elastic-stack/Dockerfile

```diff
@@ -21,10 +29,22 @@ RUN \
   apt-get install -y kibana && \
 # Install Logstash
   apt-get install -y logstash && \
+# Install S3 Repository Plugin
+  /usr/share/elasticsearch/bin/elasticsearch-plugin install -b repository-s3 && \
 # Allow network access to Elasticsearch from the host
   sed -i "s/#network.host:.*/network.host: 0.0.0.0/" /etc/elasticsearch/elasticsearch.yml && \
   sed -i "s/#server.host:.*/server.host: 0.0.0.0/" /etc/kibana/kibana.yml
 
+RUN \
+# Set the repository-s3 configuration
+  chmod 0644 /etc/cron.d/elasticsearch_backup && \
+  chown -R elasticsearch:elasticsearch /var/lib/elasticsearch && \
+  service elasticsearch restart && \
+  sleep 10 && \
+  curl -XPUT 'localhost:9200/_snapshot/s3_backup?pretty' -H 'Content-Type: application/json' \
```

Comment:
As previously said, I think that configuring this here is adding complexity and is against the Docker best practices. :/
Can't we just run this command at the container start, only if the elastic stack has never been setup?